Base.pm

```diff
@@ -1169,13 +1169,15 @@ Key = API metadata element name
 sub fieldmap {
     return {
         title => {
+            exclude   => 1,
```

Comment:
Hmm, I wonder if perhaps instead of excluding these and then adding template blocks for them explicitly, we should update the template block in the loop to allow passing of an 'input type' which we could bind on to create the right input?

Honestly.. I don't know how often such a change might come up though.. so it may well not be worth the extra code.

Reply:
Yes, I agree, that would likely be ideal.
I decided to KISS it and just follow the pattern used in RapidILL, see "type":
https://github.com/PTFS-Europe/koha-ill-rapidill/blob/master/Base.pm#L1173
https://github.com/PTFS-Europe/koha-ill-rapidill/blob/master/opac-includes/create.inc#L41

Reply:
Good enough president for me, merged... we could look at expanding the general handling more later.. I'm surprised not to see anything special regards 'array' type input actually.